ASM/c/item_table.c

```diff
@@ -328,6 +328,8 @@ item_row_t item_table[] = {
     [0x0117] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x00A1, 0x01A1, 0x88, no_upgrade, give_small_key, CASTLE_ID, -1, resolve_text_small_keys), // Ganon's Castle Small Key
     [0x0118] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x913E, 0x01A2, 0x89, no_upgrade, give_small_key, TCG_ID,    -1, resolve_text_small_keys_cmg), // Small Key (Chest Game)
 
+    [0x0119] = ITEM_ROW(0x53, HEART_CHEST_SMALL, 0x41, -1, 0x90B4, 0x0177, 0x95, no_upgrade, full_heal, -1, -1, NULL), // Fairy
```

Comment:
```suggestion
    [0x0119] = ITEM_ROW(0x53,       BROWN_CHEST, 0x41, -1, 0x90B4, 0x0177, 0x95, no_upgrade, full_heal, -1, -1, NULL), // Fairy
```

Heart chests are for items that matter for heart wincons. Recovery hearts and double defence don't use it so fairies shouldn't either.